tests/jfrogclient_test.go

```diff
@@ -13,7 +13,7 @@ import (
 
 const (
 	JfrogTestsHome      = ".jfrogTest"
-	JfrogHomeEnv        = "JFROG_CLI_HOME"
+	JfrogHomeEnv        = "JFROG_CLI_HOME_DIR"
```

Comment:
The jfrog-client-go library should not be aware or familiar with jfrog-cli. This env var and its usage should probably removed.
My guess is that this is a leftover from the time jfrog-client-go was is part of jfrog-cli.